Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM nginx:1.21.0-alpine as production
 # Copy built assets from builder
-COPY . /usr/share/nginx/html
+COPY ./src /usr/share/nginx/html
```

Comment:
`.` means current directory. 
`./src` means the directory named `src` under the current directory.
This means only `./src` is exposed to the internet.

Reply:
Ace. So you have already moved it here? I don't need to action anything?

Reply:
Correct. It's all merged. 
This is mostly an FYI if you see your files have moved